exonum/src/blockchain/schema.rs

```diff
@@ -367,7 +364,6 @@ impl IndexOwner {
 #[repr(u16)]
 enum IndexTag {
     Core = 0,
-    Dispatcher = 1,
```

Comment:
Some cleanup should be welcome. In general, this API looks too difficult; I'd try to simplify it.